apps/studio/components/interfaces/DataWarehouse/CreateWarehouseAccessToken.tsx

```diff
@@ -0,0 +1,69 @@
+import { set } from 'lodash'
+import React from 'react'
+import { Button, Input, Modal } from 'ui'
+import { FormItemLayout } from 'ui-patterns/form/FormItemLayout/FormItemLayout'
+
+type Props = {
+  onSubmit: (values: { description: string }) => Promise<void>
+}
+
+const CreateWarehouseAccessToken = (props: Props) => {
+  const [open, setOpen] = React.useState(false)
+  const [loading, setLoading] = React.useState(false)
+  const [description, setDescription] = React.useState('')
```

Comment:
can we use react hook form here? even though its only one field - but for consistency too